usage-examples/test-execution.md

```diff
@@ -0,0 +1,42 @@
+# Test Execution Example
+This example presents how Eiffel events in the test domain can be connected.
```

Comment:
I think this can be a bit more verbose. And I don't like "test domain", which implies that the protocol is somehow split into domains.